packages/paste-core/components/tooltip/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@twilio-paste/tooltip",
+  "version": "0.0.0",
+  "category": "interaction",
+  "status": "alpha",
+  "description": "Twilio's Tooltip component.",
```

Comment:
```suggestion
  "description": "The Tooltip is a popup that displays clarifying text related to an element that has been focused or hovered on.",
```

Reply:
In general, let's be more descriptive in these descriptions since they show up in search as well.